doc/INSTALL.md

```diff
@@ -206,3 +207,123 @@ Alternatively, the kpatch and kpatch-build scripts can be run directly from the
 git tree.
 
 
+### OpenEuler
```

Comment:
I think the whole "OpenEuler" section needs to go a bit further up in the file, before the "Build" and "Install" sections (they are general sections and not specific to Gentoo).

Reply:
Already fix it.